extensions/resource-deployment/src/ui/toolsAndEulaSettingsPage.ts

```diff
@@ -109,6 +122,13 @@ export class ToolsAndEulaPage extends ResourceTypePage {
 				columns: [toolColumn, descriptionColumn, installStatusColumn, versionColumn, minVersionColumn, installedPathColumn],
 				width: tableWidth,
 				ariaLabel: loc.requiredToolsText
+			}).withValidation((component) => {
```

Comment:
@Charles-Gagnon, this validation does not work. The function works and returns the value as expected but the next button does not get disabled it the function returns false. 

Also, the function is called every time there is change to the table data.

Reply:
@Charles-Gagnon @ranasaria, the overall code works without this method. If I cannot get this working then I will remove it from the PR and just fix the issue at hand

Reply:
As long as you're not regressing behavior and the next button is disabled correctly when there's still tools to be installed then removing it is fine. It's possible there's a bug with the table validation since that isn't something we validate often.

Reply:
Since this is not working, remove it. You can file a new bug into our backlog for this api not working.